google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -3321,6 +3532,30 @@ private boolean canCreateSession() {
     }
   }
 
+  private void maybeCreateMultiplexedSession() {
+    synchronized (lock) {
+      if (!numMultiplexedSessionsBeingCreated) {
+        logger.log(Level.FINE, String.format("Creating multiplexed sessions"));
+        try {
+          numMultiplexedSessionsBeingCreated = true;
```

Comment:
nit: rename this field to `multiplexedSessionBeingCreated`

Reply:
Done